src/libpriv/rpmostree-importer.cxx

```diff
@@ -627,10 +627,12 @@ import_rpm_to_repo (RpmOstreeImporter *self, char **out_csum, char **out_metadat
 
       if (!glnx_mkdtemp ("rpm-ostree-import.XXXXXX", 0700, &tmpdir, error))
         return FALSE;
-      if (!glnx_shutil_mkdir_p_at (tmpdir.fd, "usr/lib/tmpfiles.d", 0755, cancellable, error))
+      if (!glnx_shutil_mkdir_p_at (tmpdir.fd, "usr/lib/rpm-ostree/tmpfiles.d", 0755, cancellable,
+                                   error))
         return FALSE;
       if (!glnx_file_replace_contents_at (
-              tmpdir.fd, glnx_strjoina ("usr/lib/tmpfiles.d/", "pkg-", pkg_name.c_str (), ".conf"),
+              tmpdir.fd,
+              glnx_strjoina ("usr/lib/rpm-ostree/tmpfiles.d/", pkg_name.c_str (), ".conf"),
```

Comment:
One thing that will be confusing (and slightly problematic) is that today we don't invalidate the cache when the importer logic changes.  And so previously imported packages will still have the old path until they stop being cached.

I've been bitten by this in coreos-assembler builds.

Maybe in this case it's OK because we still handle the old semantics.

Reply:
Yes, that is a problem, how about checking if there is old path like `usr/lib/tmpfiles.d/pkg-{pkg_name}.conf`, then move it to `usr/lib/rpm-ostree/tmpfiles.d/{pkg_name}.conf` in earlier of `deduplicate_tmpfiles_entries()`?